packages/aic/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@adxp/aic",
+  "version": "0.0.1",
+  "main": "src/index.ts",
+  "license": "MIT",
+  "scripts": {
+    "start": "node dist/index.js",
+    "test": "jest",
+    "prettier": "prettier --check src/",
+    "prettier:fix": "prettier --write src/",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:fix": "yarn lint --fix",
+    "verify": "run-p prettier lint",
+    "verify:fix": "yarn prettier:fix && yarn lint:fix",
+    "build": "node ./build.js",
+    "postbuild": "tsc --build tsconfig.build.json",
+    "low": "node dist/low_pid.js"
+  },
+  "dependencies": {
+    "@adxp/common": "*",
```

Comment:
remove `common` if we're not using

looks like we're keeping `crypto` in here for now? I like that for now 👍 
Once we break up the `common` package a bit, we might be able to pull in just the TID part without the cruft